### PR TITLE
[p5.js hi] Missing  Maths References Added

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
-fract.mdx
-log.mdx
-mag.mdx
-noiseSeed.mdx
-randomGaussian.mdx
-random.mdx
-randomSeed.mdx
-sq.mdx

### CODE REVIEW

File: src/content/reference/hi/p5/log.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    let myNum = 5;
+    console.log(myNum); // prints 5 to the console
+    console.log(myNum + 12); // prints 17 to the console
+    </code>
+    </div>
+class: console
+params:
+  - name: message
+    description: |
+      <p>संदेश जिसे आप कंसोल पर प्रिंट करना चाहते हैं</p>
```
Comment (TanviKumar) [inline src/content/reference/hi/p5/log.mdx:36]: Instead of prompt maybe something else can be used?

File: src/content/reference/hi/p5/log.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+module: नींव
+submodule: नींव
+file: src/core/reference.js
+description: >
+    <p>आपके ब्राउज़र के वेब कंसोल पर एक संदेश प्रिंट करता है। p5 का उपयोग करते समय, आप <a href='/reference/p5/print'>print</a> और 
+    <a href='/reference/p5/console/log'>console.log</a> का परस्पर उपयोग कर सकते हैं। </p>
+
+    <p>आप जिस ब्राउज़र का उपयोग कर रहे हैं उसके आधार पर कंसोल अलग-अलग तरीके से खोला जाता है। यहां 
+    <a href="https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opending_the_Web_Console">फ़ायरफ़ॉक्स</a>, 
+    <a href="https) में कंसोल खोलने के तरीके के लिंक दिए गए हैं ://developers.google.com/web/tools/chrome-devtools/open">Chrome</a>,
```
Comment (TanviKumar) [inline src/content/reference/hi/p5/log.mdx:12]: Check the anchor tags in this file

File: src/content/reference/hi/p5/mag.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
+---
+title: mag
+module: गणित
+submodule: गणना
+file: src/math/calculation.js
+description: >
+  <p>एक वेक्टर के परिमाण, या लंबाई की गणना करता है।</p>
+
+  <p>एक वेक्टर के बारे में विभिन्न तरीकों से सोचा जा सकता है। एक दृष्टिकोण में, एक वेक्टर अंतरिक्ष में एक बिंदु है। वेक्टर के घटक,
```
Comment (TanviKumar) [inline src/content/reference/hi/p5/mag.mdx:9]: Is Antriksh correct usage here, do check once

File: src/content/reference/hi/p5/random.mdx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
...or bear.');\n}\n</code>\n</div>\n\n<div>\n<code>\nfunction setup() {\n  createCanvas(100, 100);\n\n  // Slow the frame rate.\n  frameRate(5);\n\n  describe('A black dot moves around randomly on a gray square.');\n}\n\nfunction draw() {\n  background(200);\n\n  // Get random coordinates between 0 and 100.\n  let x = random(100);\n  let y = random(100);\n\n  // Draw the point.\n  strokeWeight(5);\n  point(x, y);\n}\n</code>\n</div>\n\n<div>\n<code>\nfunction setup() {\n  createCanvas(100, 100);\n\n  // Slow the frame rate.\n  frameRate(5);\n\n  describe('A black dot moves around randomly in the middle of a gray square.');\n}\n\nfunction draw() {\n  background(200);\n\n  // Get random coordinates between 45 and 55.\n  let x = random(45, 55);\n  let y = random(45, 55);\n\n  // Draw the point.\n  strokeWeight(5);\n  point(x, y);\n}\n</code>\n</div>\n\n<div>\n<code>\nlet x = 50;\nlet y = 50;\n\nfunction setup() {\n  createCanvas(100, 100);\n\n  background(200);\n\n  describe('A black dot moves around randomly leaving a trail.');\n}\n\nfunction draw() {\n  // Update x and y randomly.\n  x += random(-1, 1);\n  y += random(-1, 1);\n\n  // Draw the point.\n  point(x, y);\n}\n</code>\n</div>"
```
Comment (TanviKumar) [inline src/content/reference/hi/p5/random.mdx:11]: Issues with alignment here

File: src/content/reference/hi/p5/randomSeed.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+title: randomSeed
+module: गणित
+submodule: यादृच्छिक
+file: src/math/random.js
+description: >
+  <p> <a href='/reference/p5/random'>random()</a> और <a href='/reference/p5/randomGaussian'>randomGaussian()</a> 
+  के लिए बीज मान सेट करता है कार्य.</p>
+
+  <p>डिफ़ॉल्ट रूप से, <a href='/reference/p5/random'>random()</a> और <a href='/reference/p5/randomGaussian'>randomGaussian()</a> 
+  अलग-अलग परिणाम देते हैं हर बार एक स्केच चलाया जाता है। <code>randomSeed()</code> को निरंतर तर्क के साथ कॉल करने से, जैसे कि
```
Comment (TanviKumar) [inline src/content/reference/hi/p5/randomSeed.mdx:11]: Can this be rephrased? The grammar looks off

**Review by TanviKumar — CHANGES_REQUESTED** [review]

File: src/content/reference/hi/p5/log.mdx (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply src/content/reference/hi/p5/log.mdx:12]: fixed

File: src/content/reference/hi/p5/mag.mdx (hunk 3 above)
Comment (SanjaySinghRajpoot) [reply src/content/reference/hi/p5/mag.mdx:9]: changed with jagha

File: src/content/reference/hi/p5/randomSeed.mdx (hunk 5 above)
Comment (SanjaySinghRajpoot) [reply src/content/reference/hi/p5/randomSeed.mdx]: done

File: src/content/reference/hi/p5/log.mdx (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply src/content/reference/hi/p5/log.mdx:35]: Can you please elaborate on this?

File: src/content/reference/hi/p5/random.mdx (hunk 4 above)
Comment (SanjaySinghRajpoot) [reply src/content/reference/hi/p5/random.mdx:11]: looking into this

**Review by TanviKumar — APPROVED** [review]

**Comment by TanviKumar** [comment]
@divyansh013 PTAL

**Review by Divyansh013 — APPROVED** [review]

**Comment by TanviKumar** [comment]
@limzykenneth this PR can be merged

**Pull request merged**